app/static/js/trackVolunteers.js

```diff
@@ -11,6 +11,18 @@ $(document).ready(function() {
     }
   });
 
+  var table =  $('#trackOutsideParticipants').DataTable({
+  "fnDrawCallback": function(oSettings) {
+    if ($('#trackOutsideParticipants tr').length < 11) {
+        $('.dataTables_paginate').hide(); //disable search and page numbers when the length of the table is less 11
```

Comment:
This comment should be above the whole block, so I don't have to read it to know what it does.